cpp/test/prims/batched/batched_matrix.cu

```diff
@@ -23,11 +23,12 @@
 
```

Comment:
nitpick: since this file is already under `batched` folder, maybe just call this one as `matrix.cu`?